gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem

```

Comment:
Similarly to #32133, you should not add `gradlew.bat` files to this PR (please don't `git add --all` otherwise you will add those files).

Reply:
yes doing a git add <filename> would fix this, but wasnt sure as it appeared after i took a pull form the main branch. so shall i reverse the commit and only push the other relevant files? @cortinico . Not too sure with the exact reason why this kept showing up after the pull.

Reply:
https://github.com/fabriziobertoglio1987/react-native-notes/issues/8#issuecomment-1043856916